packages/crypto/test/hmac.test.ts

```diff
@@ -0,0 +1,48 @@
+import { getBytes } from '@fuel-ts/utils';
+
+import { computeHmac } from '..';
+
+// #TODO: This computes the wrong value on node yet the other tests pass, I'm not sure why
+/**
+ * @group browser
+ */
+describe('computeHmac node & browser', () => {
+  it('should compute HMAC correctly', () => {
+    const key = '0x0102030405060708090a0b0c0d0e0f10';
+    const data = '0x11121314151617181920212223242526';
+    const expectedSha256 = '0x5ce589e9b1295be92db7f5e08a5672e8e5f2e80163cc706a9584269f294fe3bd';
```

Comment:
To my understanding these are the correct values

based on https://cryptotools.net/hmac